### DISCUSSION TITLE
The OWTOP 10 SQLi vulnerability is a false positive   [bug ]

### DISCUSSION THREAD

**Discussion by ltfafei** [projectdiscovery/nuclei#3284] [Q&A]
False positives were found in the SQLi scans carried out by the nuclei.

The hope is to optimise the sqli scanning engine built into the nuclei

 

 
![image](https://user-images.githubusercontent.com/43526141/217135081-207229de-7622-448a-a416-cb6f06d19626.png)

**Comment by ehsandeep** [comment]
@ltfafei nuclei result is based on what you defined in the template, and it seems you are running a custom template, as such, you need to improve your template in order to avoid the false positive results, let me know if you still belive it has something to do with nuclei engine with examples to run.

**Comment by ltfafei** [comment]
Well, I don't know who wrote a FUZZ template for SQLi, it's buggy
